docs/python-configuration.md

```diff
@@ -62,19 +158,48 @@ scope:
       statement_values: true
 ```
 
+## Include HTTP payloads in HTTP span tags
+
+You can include HTTP payloads in the related HTTP spans, truncated to the first 512 bytes.
+
+By default, Scope Java agent will not send HTTP payloads as they are considered sensitive information.
+
+### Using Environment Variables
+
+```sh
+SCOPE_INSTRUMENTATION_HTTP_PAYLOADS=true
+```
+
+
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  instrumentation:
+    http:
+      payloads: true
+```
+
 ## Include additional HTTP Headers in HTTP span tags
 
 You can include additional HTTP Headers in HTTP span tags.
 
-By default, the Scope Python agent will send common HTTP headers, filtering the content of those which can contain sensitive information, such as:
+By default, Scope Java agent will send common HTTP headers, filtering the content of those which can contain sensitive information, such as:
 
 - `Authorization`
 - `Cookie`
 - `Set-Cookie`
 
+Note that the content of every header configured explicitly to be shown in the HTTP span tags will not be filtered.
```

Comment:
this is not like this in python

Reply:
I deleted this line, should you create an issue to add it?

Reply:
not sure if it is a good practice but created an issue to track it anyway https://github.com/undefinedlabs/scope-python-agent/issues/155